README.md

```diff
@@ -14,6 +14,34 @@ See the [documentation for the Maven Plugin](https://github.com/STAMP-project/bo
 
 ### Command line interface
 
+#### botsing preprocessing
+
+The latest version of Botsing command line (botsing-preprocessing-X-X-X.jar) is available at [https://github.com/STAMP-project/botsing/releases](https://github.com/STAMP-project/botsing/releases). 
+
+Botsing preprocessing has these mandatory parameters (key/value):
+ - `-i` represents the input file path (`crash_log`) with the stack trace to clean. For example `-i=path-name-of-crash-log`
```

Comment:
From the source code, I think there is not '=' between the option (`-i`) and the value (`path-name-of-crash-log`). Could you double check and update the documentation accordingly.